src/SignalR/clients/java/signalr/core/src/main/java/com/microsoft/signalr/TransferFormat.java

```diff
@@ -3,7 +3,7 @@
 
 package com.microsoft.signalr;
 
-enum TransferFormat {
```

Comment:
I know I just approved this, but can we keep TransferFormat internal? As far as I can tell `HubProtocol.getTransferFormat()`, the only thing that uses this type, is only called from tests.